soccer/planning/RRTPlanner.cpp

```diff
@@ -192,7 +215,11 @@ vector<Point> RRTPlanner::runRRT(MotionInstant start, MotionInstant goal,
     biRRT.setStepSize(*RRTConfig::StepSize);
     biRRT.setMinIterations(_minIterations);
     biRRT.setMaxIterations(_maxIterations);
```

Comment:
Right now, this runs the RRT twice for every path plan, can we bump down minIterations to 0, and decrease the max iterations, and maybe even bump up the stepsize to get the first iteration as fast as possible? I think we might even be able to get it to only take one step (which would be much faster than running it twice). 

Make sure to set these dynamically, for example, don't hard code the stepsize higher, you could just set it to the distance between the goal and the target (and maybe you could set minIterations to 2).